ownCloud/Resources/Info.plist

```diff
@@ -35,6 +35,19 @@
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
 	<string>$(APP_SHORT_VERSION)</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Viewer</string>
+			<key>CFBundleURLName</key>
+			<string>com.owncloud.com</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>owncloud</string>
```

Comment:
Nice, that we have a url scheme to open a resource from outside. Is it also possible to open a file without the complete https url?
I would like to open a file like that:
using fileID: `owncloud://f/22` 
using localID: `owncloud://lid//76CC18...`

Reply:
@hosy would be possible, but that's the question if we shall leave this feature in e.g. for AppStore customers? cc: @michaelstingl

Reply:
> we should build the app in a way both methods are supported. (magic `owncloud://` links as well as `https://cloud.domain.com/f/12345` in branded app)

(from https://github.com/owncloud/enterprise/issues/3551#issuecomment-537088097)

I hope we can get a server extension that converts private links to magic links when opened on mobile devices. This could boost the install base – that's why I would keep it free. (see https://github.com/owncloud/enterprise/issues/3660)

Reply:
I think it would make our app more powerful to open files or folders from outside. You can open a file directly from Shortcuts.app or from other apps.